### ISSUE TITLE
Raspi HAT "waiting for connection"

### ISSUE THREAD

**Issue by MoorPoor** [NebraLtd/AnyBeam#28]
Hei.
finally I found a good usecase for the RaspiHAT.
The moment everything works it asks me to connect a device.
The display comes pretty much immediately after plugging in the power source.
I have never seen this in all the many past boots.

is it possible that the HAT needs to have its firmware reloaded in some way?
before that it ran successfully with the Raspi 4
I tried it again with a Raspi3, without success

![photo_2022-03-31_10-47-05](https://user-images.githubusercontent.com/60827766/161015741-8b77e2ff-9ab9-409e-8384-21bfa42bc5c7.jpg)

**Comment by ChristopherRush** [comment]
@MoorPoor I'm assuming you did run the install script right? Check everything was configured in the config.txt file.

**Comment by MoorPoor** [comment]
no, I configured the config file by hand, and it was running and projecting well.
but suddenly no longer after a restart. without any changes
